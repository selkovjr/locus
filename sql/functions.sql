--
--  Locus datatype test
--
-- Testing general-purpose functions
--
select locus_inter('1:152280770-152280782', '1:152280773-152280785');
select locus_union('1:152280781-152280782', '1:152280773-152280785');
