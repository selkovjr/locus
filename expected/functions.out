--
--  Locus datatype test
--
-- Testing general-purpose functions
--
select locus_inter('1:152280770-152280782', '1:152280773-152280785');
      locus_inter
-----------------------
 1:152280773-152280782
(1 row)

select locus_union('1:152280781-152280782', '1:152280773-152280785');
      locus_union
-----------------------
 1:152280773-152280785
(1 row)

